doc/WebRTC.xml

```diff
@@ -48,6 +48,14 @@
         </author>
         <revremark>First release.</revremark>
       </revision>
+      <revision>
+      <revnumber>25.06</revnumber>
+        <date>June, 2025</date>
+        <author>
+          <personname>Tomasz Zajac</personname>
+        </author>
+        <revremark>Added data channels.</revremark>
+      </revision>

```

Comment:
Hi Tomasz,
      just want to highlight these lines as they specify at the start of this document that the use of Data Channels are is outside of the scope of this version of the specification.

Can I please ask you update the same also. I have reviewed the content and was going to approve the same but for these sentences.

https://github.com/onvif/specs/blob/94b5e2c0f5048819f5d775460573a0dd78428360/doc/WebRTC.xml#L66-L68

Reply:
Thank you Kieran, I have updated this paragraph and also removed mentions of directory protocol, which I believe is no longer valid.